File: phyton/program/VoiceRecording.py
class VoiceRecording:
    # konstruktor dengan parameter
    def __init__(self, speaker, duration):
        self.speaker = speaker
        self.duration = duration
    
    # getter dan setter untuk speaker
    def setSpeaker(self, speaker):
        self.speaker = speaker
    
    def getSpeaker(self):
        return self.speaker
    
    # getter dan setter untuk duration
    def setDuration(self, duration):
        self.duration = duration
    
    def getDuration(self):
        return self.duration

File: cpp/program/VoiceRecording.cpp
#include <bits/stdc++.h>

using namespace std;

// kelas voice recording
class VoiceRecording
{
private:
    // atribut-atribut private
    string speaker;
    int duration;

public:
    // konstruktor kosong
    VoiceRecording()
    {
    }

    // konstruktor dengan parameter untuk mengisi atribut
    VoiceRecording(string speaker, int duration)
    {
        this->speaker = speaker;
        this->duration = duration;
    }

    // setter speaker
    void setSpeaker(string speaker)
    {
        this->speaker = speaker;
    }

    // getter speaker
    string getSpeaker()
    {
        return speaker;
    }

    // setter duration
    void setDuration(int duration)
    {
        this->duration = duration;
    }

    // getter duration
    int getDuration()
    {
        return duration;
    }

    ~VoiceRecording()
    {
    }
};